README.md

```diff
@@ -47,17 +50,31 @@ Details dazu finden sich [auf der folgenden Seite](docs/erp_validation.adoc).
 
 Wie Vergleiche zwischen FHIR Profilen (z.B. nach Versionsübergängen) vorgenommen werden können und den Verweis auf die Artefakte zum aktuellen Versionsübergang finden sich [auf dieser Seite](docs/erp_fhirversion_changes.adoc).
 
-## FHIR-Beispiele
+## Implementierungsunterstützung
+
+### FHIR-Beispiele
 Für die Implementierung des E-Rezeptes stehen Beispiele im FHIR-Format zur Verfügung.
 
 Im [gemeinsamen Beispiel-Repository](https://github.com/gematik/eRezept-Examples) der Gesellschafter sind sowohl Einzelbeispiele von Profilen, wie auch Beispiele von Ende zu Ende Szenarien dokumentiert.
 
 Wir wünschen uns an dieser Stelle auch ein reges Mitwirken der Industrie bei der Erstellung dieser Beispiele. Über Pull-Requests und andere Formen der Kollaboration freuen wir uns.
 
+### Konvertierung von FHIR XML und JSON
+Der FHIR Standard unterstützt für den Datenaustausch mehrere Formate. Die beiden vom E-Rezept Fachdienst unterstützten Formate sind XML (Content-Type: application/fhir+xml) und JSON (Content-Type: application/fhir+json). Der Fachdienst unterstützt an jedem Endpunkt beide Formate. Mit den Gesellschaftern wurde abgestimmt, dass bei der Kommunikation und Beschreibung der Endpunkte, die Primärsysteme betreffen, das Format XML genutzt wird. Das heißt, dass die Beispiele in der API und im [eRezept-Examples Repository](https://github.com/gematik/eRezept-Examples), die die Primärsysteme betreffen in XML dargestellt werden.
+Der Datenaustausch zwischen dem Fachdienst und dem Frontend des Versicherten (FdV) dagegen geschieht im JSON-Format.
+
+Folgende Tools können genutzt werden, um FHIR-Dokumente zwischen XML und JSON zu konvertieren:
+* [Webseite zum Konvertieren](https://fhir-formats.github.io/)
```

Comment:
Ursprünglicher Gedanke war das "Swiss Army Knife for FHIR" zur Verfügung zustellen, welches sowohl Example-Instanzen erzeugen kann als auch die Möglichkeit bietet Ressourcen zu konvertieren. Ich finde den Ansatz jetzt weitere Alternativen aufzuzeigen sehr gut. Welche der Varianten konnten getestet werden?

Reply:
Was ist das Swiss Army Knife for FHIR`? 
Die ersten 3 habe ich selbst getestet/ gesehen. Das letzte kommt von HAPI FHIR, da habe ich vertrauen :) 

Kann das so raus?

Reply:
Ja, gute Arbeit!